src/components/fields/field-datepicker/field-datepicker-pure.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC } from 'react';
+import get from 'lodash/get';
```

Comment:
Ikke nødvendig her. field, touched og errors kan destrukteres for å hente nødvendige verdier gitt riktige typ deklarasjoner

Reply:
Egentlig trenger jeg ikke touched og errors her i det hele tatt som jeg kan se.